docs/In-Band Manageability User Guide - INBS.md

```diff
@@ -45,3 +45,37 @@ To disable TLS (for example, for testing), set "tls_enabled" to false. tls_cert_
 - **tls_enabled**: Boolean value to enable or disable TLS. Set to `true` to enable TLS.
 - **tls_cert_path**: (ignored if TLS is disabled) The system path to the TLS certificate used for establishing a secure connection to the INBS server. It is recommended that the path be under `/etc/intel-manageability/public/` to ensure the certificate is accessible to the cloudadapter agent.
 - **token_path**: (ignored if TLS is disabled) Path to the file containing the confidential authentication token used to verify the device with the INBS server. It is recommended that the path be under `/etc/intel-manageability/secret/` to ensure the token is secure.
+
+### UDM Bootstrap Mode
+
+The user can also set up `adapter.cfg` in "UDM Bootstrap" mode, like this:
+
+```
+{
```

Comment:
Are we supporting both forms of adapter.cfg content in UDM mode? i.e. would the below also work?

```json
{
  "cloud": "inbs",
  "config": {
    "hostname": "localhost",
    "port": "5678",
    "node-id": "abcdefg",
    "tls_enabled": true,
    "tls_cert_path": "/path/to/your/certificate.pem",
    "token_path": "/path/to/your/token.txt"
  }
}
```

Reply:
> Are we supporting both forms of adapter.cfg content in UDM mode? i.e. would the below also work?

This will still work.